site-manage/datemanager/impl/src/main/java/org/sakaiproject/datemanager/impl/DateManagerServiceImpl.java

```diff
@@ -1244,7 +1244,13 @@ public JSONArray getForumsForContext(String siteId) {
 					tobj.put(DateManagerConstants.JSON_DUEDATE_PARAM_NAME, null);
 					tobj.put(DateManagerConstants.JSON_OPENDATE_PARAM_NAME, null);
 				}
-				tobj.put(DateManagerConstants.JSON_EXTRAINFO_PARAM_NAME, resourceLoader.getString("itemtype.topic"));
+				
+				String extraInfo = resourceLoader.getString("itemtype.topic");
+				if (topic.getDraft()) {
+					extraInfo = extraInfo + " " + resourceLoader.getString("itemtype.draft"); 
```

Comment:
extraInfo should not be only the resourceLoader.getString("itemtype.draft");?
As here:  https://github.com/sakaiproject/sakai/blob/4da573e4d186563f09a568d4e58c3d2caca32bbc/site-manage/datemanager/impl/src/main/java/org/sakaiproject/datemanager/impl/DateManagerServiceImpl.java#L285

Reply:
![image](https://github.com/user-attachments/assets/f7666323-6b8f-4525-90a1-ffcfc7e13d1c)

Reply:
Oh I see. Topic Draft and Forums Draft